Build/PackageFiles/ExtensionBundleTemplates-1.x.nuspec

```diff
@@ -29,6 +29,11 @@
     <file src="../../Functions.Templates/templates/QueueTrigger-CSharp/QueueTriggerCSharp.csx" target="templates/QueueTrigger-CSharp/run.csx" />
     <file src="../../Functions.Templates/templates/QueueTrigger-CSharp/readme.md" target="templates/QueueTrigger-CSharp/readme.md" />
     <file src="../../Functions.Templates/templates/QueueTrigger-CSharp/sample.dat" target="templates/QueueTrigger-CSharp/sample.dat" />
+	<file src="../../Functions.Templates/templates/RabbitMQTrigger-CSharp/function.json" target="templates/RabbitMQTrigger-CSharp/function.json" />

```

Comment:
@soninaren Wasn't sure about the order of changes here. Maybe this PR can be checked in after adding RabbitMQ extension to bundle ? (Applies to ExtensionBundleTemplates-2.x.nuspec as well.)

Reply:
format ? extra space?